source/development/packaging/generate-dashboard-package.rst

```diff
@@ -0,0 +1,186 @@
+      # zip -r -j ./dashboard-package.zip ../wazuh-dashboard/target/opensearch-dashboards-<opensearch version>-linux-x64.tar.gz
+      # zip -r -j ./security-package.zip ../wazuh-dashboard/plugins/wazuh-security-dashboards-plugin/build/security-dashboards-<opensearch version>.0.zip
+      # zip -r -j ./wazuh-package.zip ../wazuh-dashboard/plugins/wazuh-check-updates/build/wazuhCheckUpdates-<opensearch version>.zip ../wazuh-dashboard/plugins/main/build/wazuh-<opensearch version>.zip ../wazuh-dashboard/plugins/wazuh-core/build/wazuhCore-<opensearch version>.zip
+
+At this point you must have 3 packages in the packages folder:
+
+-  ``dashboard-package.zip``
+-  ``security-package.zip``
+-  ``wazuh-package.zip``
+
+2. Generate the final packages, by running the script ``build-packages.sh`` in the ``dev-tools/build-packages/`` folder of the repository. 
+The script requires the following parameters:
+
+- ``-v``: Version of the package.
+- ``-r``: Revision of the package.
+- ``--deb`` or ``--rpm``: Distribution of the package.
+- ``-a``: Path to the ``wazuh-package.zip``.
+- ``-s``: Path to the ``security-package.zip``.
+- ``-b``: Path to the ``dashboard-package.zip``.
+
+.. code:: console
+
+   # cd ../wazuh-dashboard/dev-tools/build-packages/
+   # ./build-packages.sh -v <version> -r <revision> <distribution(--deb || --rpm)> -a <path to wazuh-package.zip> -s <path to security-package.zip> -b <path to dashboard-package.zip>
+
+The package will be generated in the ``output`` folder of the same directory where the script is located.
+
+
+Build with Docker image
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With this option you can create an image that has the package in tar.gz format
+and then if desired you can use the created package to generate the .deb or .rpm file.
+
+1. Clone the Wazuh Dashboard repository.
+
+.. code:: console
+
+   # git clone -b <branch> https://github.com/wazuh/wazuh-dashboard.git
+   # cd wazuh-dashboard/dev-tools/build-packages/
+
+2. Build the Docker image with the following parameters:
+
+- ``NODE_VERSION``: Node version to use in the ``.nvmrc`` file.
+- ``WAZUH_DASHBOARDS_BRANCH``: Branch of the Wazuh Dashboards repository.
+- ``WAZUH_DASHBOARDS_PLUGINS``: Branch of the Wazuh Dashboards Plugins repository.
+- ``WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH``: Branch of the Wazuh Security Dashboards Plugin repository.
+- ``OPENSEARCH_DASHBOARDS_VERSION``: Version of the OpenSearch Dashboards(you can find the version in the package.json file of the Wazuh Dashboards repository)
+- ``-t``: Tag of the image.
+
+.. code:: console
+
+   # docker build \
+   # --build-arg NODE_VERSION=<Node version> \
+   # --build-arg WAZUH_DASHBOARDS_BRANCH=<Branch of wazuh-dashboard> \
+   # --build-arg WAZUH_DASHBOARDS_PLUGINS=<Branch of the wazuh-dashboard-plugins> \
+   # --build-arg WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH=<Branch of wazuh-security-dashboards-plugin> \
+   # --build-arg OPENSEARCH_DASHBOARDS_VERSION=<Opensearch dashboard version> \
```

Comment:
We should let the users know how to obtain the OSD version

Reply:
I would add that it is under the pluginPlatform/version section, maybe an example